### PR TITLE
added : cart page, varibles, utilities updation

### PR DESCRIPTION
**Pull request by rishbitsnbytes**
Kidly review

### CODE REVIEW

**Comment by sah-shah7** [comment]
looks good, can be merged

File: Styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    margin: 2rem;
+    padding: 2rem;
+}
+
+.cart-main-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 2px solid var(--gray);
+    /* grid-column: 1 / -1; */
```
Comment (vidhi-thakur) [inline Styles/cart.css:17]: Please remove the commented code

File: Styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+.cart-products-shelf {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    gap: 1rem;
+}
+
+.main-head {
+    /* grid-column: 1 / -1; */
```
Comment (vidhi-thakur) [inline Styles/cart.css:30]: Please remove commented code

**Review by vidhi-thakur — APPROVED** [review]
Added sanity comments, other than that looks good to me

File: Pages/cart/index.html [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+                    My Cart
+                </h2>
+                <p class="price-head">
+                    Price
+                </p>
+            </div>
+            <article class="card card-horizontal product-shelf-card">
+                <div class="card-main">
+                    <div class="card-header">
+                        <img src="../../Assets/boysShoes1.jpg" alt="Boys Shoes 1" class="product-shelf-img">
```
Comment (aurobindoGupta) [inline Pages/cart/index.html:75]: please add alt

**Review by aurobindoGupta — APPROVED** [review]
nice work

**Pull request merged**